ci-operator/step-registry/ovn/conf/ipv4-internal-subnet/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
Maybe we can create a real file to include you and me in the OWNER list.

Reply:
sure, updated.